extension/src/content_scripts/simplification.js

```diff
@@ -36,38 +34,62 @@
         styles: {
             navToggle: "gpii-simplify-navToggle"
         },
+        members: {
+            nav: {
+                expander: {
+                    funcName: "{that}.locateOutOfContent",
+                    args: ["nav"]
+                }
+            }
+        },
         model: {
             simplify: false,
             showNav: false
         },
         modelListeners: {
             simplify: {
                 listener: "{that}.set",
-                args: ["{change}.value"]
+                args: ["{change}.value"],
+                excludeSource: ["init"]
             },
             showNav: {
                 listener: "{that}.toggleNav",
                 args: ["{change}.value"],
                 excludeSource: "init"
             }
         },
+        events: {
+            onAlwaysVisibleNodeAdded: null
+        },
         listeners: {
             "onCreate.toggleNav": {
                 listener: "{that}.toggleNav",
                 args: ["{that}.model.showNav"]
+            },
+            "onCreate.set": {
+                listener: "{that}.set",
+                args: ["{that}.model.simplify"]
             }
         },
         injectNavToggle: true,
+        components: {
+            observer: {
+                type: "gpii.chrome.mutationObserver",
```

Comment:
Explain why we need a mutationObserver here. Given we rendered the content that we want to make visible ourselves, how did it become invisible?

Reply:
@amb26 I've tried to answer that in this comment at the top of the file https://github.com/jobara/gpii-chrome-extension/blob/GPII-3443/extension/src/content_scripts/simplification.js#L18-L26

Essentially because a parent element has `visibility:hidden` set, we need to explicitly set it to `visibility:visible`.